src/org/labkey/test/util/TestUser.java

```diff
@@ -0,0 +1,119 @@
+package org.labkey.test.util;
+
+import org.labkey.api.collections.CaseInsensitiveHashMap;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.security.CreateUserResponse;
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.openqa.selenium.WebElement;
+
+import java.util.Map;
+
+import static org.labkey.test.WebDriverWrapper.WAIT_FOR_PAGE;
+import static org.labkey.test.util.TestLogger.log;
+
+public class TestUser
+{
+    private final WebDriverWrapper _test;
+    private final String _email;
+    private CreateUserResponse _createUserResponse;
+    private String _password;
+    private final APIUserHelper _apiUserHelper;
+    private Connection _impersionationConnection;
+
+    public TestUser(WebDriverWrapper test, String email)
+    {
+        _test = test;
+        _email = email;
+        _apiUserHelper = new APIUserHelper(_test);
+        create();
+    }
+
+    /**
+     * Creates the user immediately via the API, and stores the createUserResponse to hang on to data like
+     *
+     * @return
+     */
+    private TestUser create()
+    {
+        _createUserResponse = _apiUserHelper.createUser(_email);
+        return this;
+    }
+
+    public void deleteUser()
+    {
+        _apiUserHelper.deleteUsers(false, _email);
+    }
+
+    public Long getUserId()
+    {
+        return (Long) _createUserResponse.getUserId();
+    }
+
```

Comment:
Consider adding a method to get the user's display name.
```suggestion

    public Long getUserId()
    {
        return (Long) _createUserResponse.getUserId();
    }

    public String getDisplayName()
    {
        return _apiUserHelper.getDisplayNameForEmail(getEmail());
    }

```